src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java

```diff
@@ -44,55 +44,194 @@
+            feedbackQuestionAttributes.numberOfEntitiesToGiveFeedbackTo = numOfEntitiesToGiveFeedbackTo;
+            return this;
+        }
+
+        public Builder withShowResponseTo(List<FeedbackParticipantType> showResponseTo) {
+            feedbackQuestionAttributes.showResponsesTo =
+                    showResponseTo == null ? new ArrayList<>()
+                            : new ArrayList<>(showResponseTo);
+            return this;
+        }
+
+        public Builder withShowGiverNameTo(List<FeedbackParticipantType> showGiverNameTo) {
+            feedbackQuestionAttributes.showGiverNameTo =
+                    showGiverNameTo == null ? new ArrayList<>()
+                            : new ArrayList<>(showGiverNameTo);
+            return this;
+        }
+
+        public Builder withShowRecipientNameTo(List<FeedbackParticipantType> showRecipientNameTo) {
+            feedbackQuestionAttributes.showRecipientNameTo =
+                    showRecipientNameTo == null ? new ArrayList<>()
+                            : new ArrayList<>(showRecipientNameTo);
+            return this;
+        }
+
+        public Builder withCreatedAt(Instant createdAt) {
+            if (createdAt != null) {
+                feedbackQuestionAttributes.createdAt = createdAt;
+            }
+            return this;
+        }
+
+        public Builder withUpdatedAt(Instant updatedAt) {
+            if (updatedAt != null) {
+                feedbackQuestionAttributes.updatedAt = updatedAt;
+            }
+            return this;
+        }
+
+        public Builder withFeedbackQuestionId(String feedbackQuestionId) {
+            if (feedbackQuestionId != null) {
+                feedbackQuestionAttributes.feedbackQuestionId = feedbackQuestionId;
+            }
+            return this;
+        }
+
+        public FeedbackQuestionAttributes build() {
+            feedbackQuestionAttributes.questionDescription =
+                    SanitizationHelper.sanitizeForRichText(feedbackQuestionAttributes.questionDescription);
+            feedbackQuestionAttributes.removeIrrelevantVisibilityOptions();
+            return feedbackQuestionAttributes;
+        }
+
+        public FeedbackQuestionAttributes buildWithoutRemovingIrrelevantVisibilityOptions() {
+            feedbackQuestionAttributes.questionDescription =
+                    SanitizationHelper.sanitizeForRichText(feedbackQuestionAttributes.questionDescription);
+            return feedbackQuestionAttributes;
+        }
     }
 
     public FeedbackQuestionAttributes getCopy() {
-        return new FeedbackQuestionAttributes(this);
+        return builder()
+                .withFeedbackSessionName(feedbackSessionName)
+                .withCourseId(courseId)
+                .withCreatorEmail(creatorEmail)
+                .withQuestionMetaData(questionMetaData)
+                .withQuestionDescription(questionDescription)
```

Comment:
This copied value wasn't present in `FeedbackQuestionAttributes(FeedbackQuestionAttributes other)`. I think it's OK to add it, though.

Reply:
I suspect it was left out unintentionally.